### PR TITLE
[PROGRAMMES-6737] gallery siblings

### PR DESCRIPTION
**Pull request by tomwinnington**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6737

Link:
https://sandbox.bbc.co.uk/programmes/p06r53hq

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by Bobkidbob** [comment]
Is this [PROGRAMMES-6737](https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6737)?

File: src/Controller/FindByPid/GalleryController.php [hunk 1]
```diff
@@ -3,27 +3,44 @@
...
 
 class GalleryController extends BaseController
 {
     public function __invoke(
         Gallery $gallery,
         ImagesService $imagesService,
-        ?string $imagePid
+        ?string $imagePid,
+        ProgrammesAggregationService $programmesAggregationService
+
```
Comment (Bobkidbob) [inline src/Controller/FindByPid/GalleryController.php]: Remove this line or linter will fail.
```suggestion

```

File: src/Controller/FindByPid/GalleryController.php [hunk 2]
```diff
@@ -3,27 +3,44 @@
...
     ) {
         $this->setIstatsProgsPageType('galleries_show');
         $this->setContextAndPreloadBranding($gallery);
+        $siblingLimit = 4;
 
         $images = $imagesService->findByGroup($gallery);
         $image = $this->getFirstImage($imagePid, $images);
+        $programme= $gallery->getParent();
```
Comment (Bobkidbob) [inline src/Controller/FindByPid/GalleryController.php]: Add space after `$programme` or linter will fail.
```suggestion
        $programme = $gallery->getParent();
```

File: src/Controller/FindByPid/GalleryController.php [hunk 3]
```diff
@@ -3,27 +3,44 @@
...
+
+
         return $this->renderWithChrome('find_by_pid/gallery.html.twig', [
             'gallery' => $gallery,
+            'programme' => $programme,
             'image' => $image,
             'images' => $images,
+            'network' => $network,
+            'galleries' => $galleries,
+            'brand' => $brand
```
Comment (Bobkidbob) [inline src/Controller/FindByPid/GalleryController.php]: Add a comma or linter will fail.
```suggestion
            'brand' => $brand,
```

File: templates/find_by_pid/gallery.html.twig [hunk 4]
```diff
@@ -1,13 +1,48 @@
...
 {%- block body -%}
 
-<div class="programmes-page prog-layout__primary br-box-page">
-    <h1><span class="context">{{ ds_shared('entityContext', gallery,{
-                'include_self': false,
-            }) }} </span>{{ gallery.getTitle() }}</h1>
-</div>
+    <div class="grid-wrapper grid-wrapper--center">
+        <div class="grid 13/24@bpw2 13/24@bpe">
+            <div class="grid__inner">
```
Comment (Bobkidbob) [inline templates/find_by_pid/gallery.html.twig]: I think this HTML might have more redundant classes.

File: templates/find_by_pid/gallery.html.twig [hunk 5]
```diff
@@ -1,13 +1,46 @@
...
 {%- block body -%}
 
-<div class="programmes-page prog-layout__primary br-box-page">
-    <h1><span class="context">{{ ds_shared('entityContext', gallery,{
-                'include_self': false,
-            }) }} </span>{{ gallery.getTitle() }}</h1>
-</div>
+    <div class="grid-wrapper grid-wrapper--center">
+        <div class="grid 13/24@bpw2 13/24@bpe">
+
```
Comment (Bobkidbob) [inline templates/find_by_pid/gallery.html.twig]: Assuming this empty `div` is required, it looks good to me.

**Pull request merged**